decrypt/decrypt.go

```diff
@@ -90,16 +94,35 @@ func Decrypt(args []string) error {
 			err := decryptFile(file.Encrypted, file.Unencrypted, *privateKey)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Error decrypting file %s: %v\n", file.Encrypted, err)
+				decryptedCount--
 			}
+			decryptedCount++
 		case helpers.FileExists(file.Unencrypted):
 			fmt.Fprintf(os.Stderr, "Warning: file %s is already decrypted, skipping\n", file.Unencrypted)
 		default:
 			fmt.Printf("Decrypting file %v/%v: %s\n", i+1, numFiles, file.Encrypted)
 			err := decryptFile(file.Encrypted, file.Unencrypted, *privateKey)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Error decrypting file %s: %v\n", file.Encrypted, err)
+				decryptedCount--
 			}
+			decryptedCount++
 		}
+		// remove the encrypted file if the clean flag is set
+		if *clean && helpers.FileIsReadable(file.Encrypted) {
+			err = os.Remove(file.Encrypted)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Could not remove encrypted file %s: %s", file.Encrypted, err)
+
+				continue
+			}
+			removedCount++
+		}
+
+	}
+	fmt.Printf("Decryption completed.\n%v files decrypted\n", decryptedCount)
+	if *clean {
+		fmt.Printf("%v encrypted files removed\n", removedCount)
```

Comment:
This feels like it should be the opposite i.e. notify if files could *not* be removed.

Reply:
Right, so if it were to stay consistent to current behavior, no message for success and message for failures, either removing files or decrypting